### PR TITLE
Tidy up

### PR DESCRIPTION
**Pull request by denialanderror**
Similar to the Pro Hub one, this adds Prettier and reformats everything. Also removed the trailing slash from the root route and fixes a Docker issue.

### CODE REVIEW

**Review by ramchale — DISMISSED** [review]

File: cypress/integration/navigation.spec.js [hunk 1]
```diff
@@ -2,13 +2,16 @@ describe("Navigation bar", () => {
...
     });
 
     const expected = [
-        ["Dashboard", "/supervision/deputies/public-authority/deputy/1/"],
+        ["Dashboard", "/supervision/deputies/public-authority/deputy/1"],
         ["Clients", "/supervision/deputies/public-authority/deputy/1/clients"],
-        ["Timeline", "/supervision/deputies/public-authority/deputy/1/timeline"],
+        [
+            "Timeline",
+            "/supervision/deputies/public-authority/deputy/1/timeline",
```
Comment (MishNajam) [inline cypress/integration/navigation.spec.js:13]: is it worth removing the trailing comma and keeping this to one line for consistency?

File: docker/json-server/Dockerfile [hunk 2]
```diff
@@ -1,18 +1,3 @@
...
-RUN mkdir -p web/static
-
-COPY web/assets web/assets
-COPY webpack.config.js .
-COPY package.json .
-COPY yarn.lock .
-
-RUN yarn install
-RUN yarn build
-
```
Comment (MishNajam) [inline docker/json-server/Dockerfile:15]: for my own understanding - do we not longer need these? Have we set these up somewhere else in the repo?

**Review by MishNajam — APPROVED** [review]

File: cypress/integration/navigation.spec.js (hunk 1 above)
Comment (denialanderror) [reply cypress/integration/navigation.spec.js:13]: This is auto-formatted according to the Prettier rules. It is possible to change the suggested line length but it isn't advised apparently.

File: docker/json-server/Dockerfile (hunk 2 above)
Comment (denialanderror) [reply docker/json-server/Dockerfile:15]: Before we introduced Mux to do more advanced routing, this was necessary as the web assets were just placed on the file system in the public directory, so in order to serve them with json-server running, we had to make them available in its container. This meant duplicating the asset build process, which increased build time and complexity (albeit only locally and not by much).

Now that we have Mux and its static file routing, the app container can serve its own assets, so this part is no longer necessary.

**Pull request merged**